crates/engine/codegen/domain/schema.graphql

```diff
@@ -267,6 +269,11 @@ type VirtualSubgraph @meta(module: "subgraph/virtual_") @indexed(id_size: "u16")
   schema_directives: [TypeSystemDirective!]!
 }
 
+enum SubscriptionsProtocol @meta(module: "subgraph/subscriptions_protocol") {
+  SERVER_SENT_EVENTS
+  WEBSOCKETS
```

Comment:
Why plural for websocket?

Reply:
Just by feel. I don't care either way, happy to change to singular.